website/docs/source/v2/provisioning/ansible_common.html.md

```diff
@@ -51,6 +51,30 @@ Some of these options are for advanced usage only and should not be used unless
 
   By default, this option is disabled and Vagrant generates an inventory based on the `Vagrantfile` information.
 
+- `galaxy_command` (template string) - The command pattern used to install Galaxy roles when `galaxy_role_file` is set.
+
+  The following placeholders can be used in this command pattern:
+    - `%{ROLE_FILE}` is replaced by the absolute path to the `galaxy_role_file` option
+    - `%{ROLES_PATH}` is
+      * replaced by the absolute path to the `galaxy_roles_path` option when such option is defined
+      * replaced by the absolute path to a `roles` subdirectory sitting in the parent directory of the configured `playbook` file otherwise.
+
+  By default, this option is set to
+
+  ```
+  ansible-galaxy install --role-file=%{ROLE_FILE} --roles-path=%{ROLES_PATH} --force
```

Comment:
This is actually not an effective command, but a "command skeleton" value. I think that putting a `$` could be misleading.
The source of confusion is certainly due to my "triple-backquoting" enclosure. I'll change it for a "simple backquote" style:

`ansible-galaxy install --role-file=%{role_file} --roles-path=%{roles_path} --force`

It won't look very nice, but it is probably the clearest way to present this information.